packages/contracts-rfq/contracts/interfaces/IFastBridge.sol

```diff
@@ -24,14 +24,31 @@ interface IFastBridge {
 
     // ============ Events ============
 
-    event BridgeRequested(bytes32 transactionId, address sender, bytes request);
+    event BridgeRequested(
+        bytes32 indexed transactionId,
+        address indexed sender,
+        bytes request,
+        uint32 destChainId,
+        address originToken,
+        address destToken,
+        uint256 originAmount,
+        uint256 destAmount,
+        bool sendChainGas
+    );
     event BridgeRelayed(
-        bytes32 transactionId, address relayer, address to, address token, uint256 amount, uint256 chainGasAmount
+        bytes32 indexed transactionId,
+        address indexed relayer,
+        address indexed to,
+        address token,
+        uint256 amount,
+        uint256 chainGasAmount
```

Comment:
Let's add `originChainId` here. We would also want to have the same origin/destToken and origin/destAmount pairs being emitted here as in `BridgeRequested`